### PR TITLE
Semantics/checkbox check

### PR DESCRIPTION
**Pull request by alphapokedex**
## Changes:
- Semantics for check box 
- change log updated
- pub version bump

### CODE REVIEW

File: packages/regal/.fvmrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "flutter": "3.19.5"
```
Comment (yasht01) [inline packages/regal/.fvmrc:2]: @alphapokedex why is this getting added?

File: packages/regal/lib/src/widgets/regal_checkbox.dart [hunk 2]
```diff
@@ -49,62 +49,65 @@ class _RegalCheckboxState extends State<RegalCheckbox> {
...
-                    borderRadius: widget.circular
-                        ? BorderRadius.circular(widget.size)
-                        : BorderRadius.circular(4),
-                    border: Border.all(
-                      color: (_checked
-                              ? widget.activeColor
-                              : widget.borderColor) ??
-                          context.theme.primaryColor,
+  Widget build(BuildContext context) => Semantics(
+        checked: widget.checked,
```
Comment (yasht01) [inline packages/regal/lib/src/widgets/regal_checkbox.dart:53]: @alphapokedex is a label needed here?

**Review by yasht01 — CHANGES_REQUESTED** [review]

File: packages/regal/lib/src/widgets/regal_checkbox.dart (hunk 2 above)
Comment (alphapokedex) [reply packages/regal/lib/src/widgets/regal_checkbox.dart:53]: nope, already in child widget

File: packages/regal/.fvmrc (hunk 1 above)
Comment (alphapokedex) [reply packages/regal/.fvmrc:2]: to support fvm

**Review by yasht01 — APPROVED** [review]

**Review by omer-vancha — APPROVED** [review]

**Review by vamsi-kleargroup — APPROVED** [review]

**Pull request merged**